### PR TITLE
[Relay] [Frontend] Add support for MXNet smooth_l1

### PR DESCRIPTION
**Pull request by haojin2**
...in Relay frontend based on existing operations.

### CODE REVIEW

File: python/tvm/relay/frontend/mxnet.py [hunk 1]
```diff
@@ -566,6 +566,16 @@ def _mx_embedding(inputs, _):
     return _op.take(weight, indices.astype('int32'), axis=0)
 
 
+def _mx_smooth_l1(inputs, attrs):
+    scalar = attrs.get_float("scalar", 1.0)
+    scalar_sq = scalar * scalar
+    print(type(inputs[0]))
```
Comment (yzhliu) [inline python/tvm/relay/frontend/mxnet.py]: remove

Comment (haojin2) [reply python/tvm/relay/frontend/mxnet.py]: Done

**Comment by haojin2** [comment]
@tqchen This version good enough from your perspective?

**Comment by tqchen** [comment]
Will leave it to @yzhliu for comments. Note there might be regressions when we change the data types, because the constants are still f32. But it might be fine for now

**Comment by yzhliu** [comment]
Thanks all. this is merged.

**Pull request merged**